docs/advanced-configuration.md

```diff
@@ -103,6 +139,7 @@ There are following known limitations of native OTel logs collection:
   This means that correlation between logs and traces will not work in Splunk Observability.
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
+- Not yet supported in GKE Autopilot.
```

Comment:
way want to use bold or place in a more disclaimer-ish position near the distribution description.

Reply:
This is a short term limitation. Will be addressed in January. But I'll add the same note to the Autopilot section

Reply:
added